src/components/Transaction/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import ProgressIndicator from 'components/ProgressIndicator'
+import moment from 'moment'
+
+import { RESOLUTION_TIME, TRANSACTION_COMPLETE_STATUS } from 'utils/constants'
+
+const completionMessages = {
+  [TRANSACTION_COMPLETE_STATUS.NO_ERROR]: 'Transaction finished successfully',
+  [TRANSACTION_COMPLETE_STATUS.ERROR]: 'Transaction did not finish, errors occured',
+  [TRANSACTION_COMPLETE_STATUS.TIMEOUT]: 'Transaction timed out',
+  [TRANSACTION_COMPLETE_STATUS.LOST]: 'Because of a page reload, we lost the status of this Transaction',
+}
+
+const Transaction = ({ label, startTime, endTime, completed, completionStatus, progress, type }) => (
+  <div className={`transactionsPage__transaction transactionsPage__transaction--${type} transaction`}>
+    <ProgressIndicator completed={completed} completionStatus={completionStatus} progress={progress} />
+    <div className="transaction__content">
+      <div className="transaction__heading">{label}</div>
+      <div className="transaction__details">
+        <div className="transaction__detail">
```

Comment:
I would create every *transaction__detail* into a separate PRIVATE component (so place them as constants in this file). As far as I see there are three different:
* Created at
* Finished at
* Transaction time.

Suggestions:
I think those components could be reused/refactored into one, the differences between them are the text and the children props.